genericjpa/async-updates/src/main/java/org/hibernate/search/genericjpa/jpa/util/impl/JPAEntityManagerFactoryWrapper.java

```diff
@@ -0,0 +1,29 @@
+package org.hibernate.search.genericjpa.jpa.util.impl;
```

Comment:
It looks like these classes are missing the license header.. which makes me think that somehow you got checkstyle disabled here as it would have failed the build and pointed it out.
Could you please make sure that checkstyle is run on your modules too? It might find some formatting issues too.. I hope not too many to be a PITA but it's certainly nicer to send the initial contribution with the right formatting than having to fix it later.